x-pack/plugins/elastic_assistant/server/lib/langchain/elasticsearch_store/elasticsearch_store.ts

```diff
@@ -157,6 +157,7 @@ export class ElasticsearchStore extends VectorStore {
     try {
       const response = await this.kbDataClient.addKnowledgeBaseDocuments({
         documents,
+        global: true,
```

Comment:
This API isn't used anymore, but I updated it just in case someone inadvertently uses it in the interim.